src/sema/ast.rs

```diff
@@ -181,8 +181,8 @@ pub struct EventDecl {
 impl EventDecl {
     pub fn symbol_name(&self, ns: &Namespace) -> String {
         match &self.contract {
-            Some(c) => format!("{}.{}", ns.contracts[*c].id, self.name),
-            None => self.name.to_string(),
+            Some(c) => format!("{}.{}", ns.contracts[*c].id, self.id),
+            None => self.id.to_string(),
```

Comment:
No test for this line.

Reply:
How do we fix the coverage bot so we don't get failure for this?

If not, we have to turn the coverage bot off, because otherwise development will grind to a halt.

Reply:
I've submitted https://github.com/hyperledger/solang/pull/1585